### PR TITLE
Update phoenix to 1.3

### PR DESCRIPTION
**Pull request by nunopolonia**
This PR only addresses making the app work with Phoenix 1.3.

There is still some work needed to make this a 1.3 application. The models should disappear and the app should be separated by contexts but I feel this is a good start.

### CODE REVIEW

File: priv/repo/migrations/20170730233801_add_role_to_user.exs [hunk 1]
```diff
@@ -12,6 +12,6 @@ defmodule Api.Repo.Migrations.AddRoleToUser do
 
     flush()
 
-    from(u in "users", update: [set: [role: "participant"]]) |> Api.Repo.update_all([])
+    from(u in "users", update: [set: [role: "participant"]]) |> ApiWeb.Repo.update_all([])
```
Comment (pkoch) [inline priv/repo/migrations/20170730233801_add_role_to_user.exs]: Why not comment out altogether? That seems to be where data migrations end up, anyway.

**Comment by pkoch** [comment]
What happened to the CircleCI build?

File: priv/repo/migrations/20170730233801_add_role_to_user.exs (hunk 1 above)
Comment (nunopolonia) [reply priv/repo/migrations/20170730233801_add_role_to_user.exs]: I only commented the ones that failed. But you're right, it makes sense to do it.

**Comment by nunopolonia** [comment]
@pkoch What do you mean, what happened to the CircleCI build?

**Comment by pkoch** [comment]
Oh, derp! I couldn't find it on the desktop webui. :| My bad!

**Pull request merged**